app/uk/gov/hmrc/helptosavefrontend/connectors/NSIConnector.scala

```diff
@@ -67,6 +69,10 @@ class NSIConnectorImpl @Inject()(conf: Configuration) extends NSIConnector {
       .map { response ⇒
         response.status match {
           case Status.CREATED ⇒
+            val applicationEventResult = auditConnector.sendEvent(new ApplicationSubmittedEvent(userInfo))
+            applicationEventResult.onFailure {
+              case e: Throwable => Logger.error(s"Unable to post application submission event to audit connector - ${e.getMessage}", e)
```

Comment:
It just logs the message. One of the rules in MDTP best practice for logging is never, ever log a stack trace.